src/templates/about-page.js

```diff
@@ -87,10 +87,9 @@ export const AboutPageTemplate = ({
                     </h2>
                   </div>
                   <div className="column is-12">
-                    <Link className="button has-text-centered"
-                      style={{fontWeight: 'bold', fontSize: '20px'}} size="large" href="/">
+                    <button class="button is-primary-invert is-large">
```

Comment:
Why is this _not_ a `<Link to=...`? It should just be a link to the Create a Project page.

- [ ] Make this button link to the Create a Project page.